tycho-extras/tycho-extras-its/src/test/resources/mirrortest-from-target-platform/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2022 Mat Booth. This program and the accompanying materials 
+	are made available under the terms of the Eclipse Public License 2.0 which 
+	accompanies this distribution, and is available at https://www.eclipse.org/legal/epl-2.0/ 
+	SPDX-License-Identifier: EPL-2.0 -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.example</groupId>
+	<artifactId>mirrortest</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<packaging>eclipse-repository</packaging>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-p2-extras-plugin</artifactId>
+					<version>${tycho-version}</version>
+					<configuration>
+						<ius>
+							<iu>
+								<id>org.mockito.mockito-core</id>
+							</iu>
+						</ius>
+						<includeOptional>false</includeOptional>
+						<includeNonGreedy>false</includeNonGreedy>
+						<latestVersionOnly>true</latestVersionOnly>
+						<compress>false</compress>
+					</configuration>
+				</plugin>
+			</plugins>
+		</pluginManagement>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<extensions>true</extensions>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<version>${tycho-version}</version>
+				<configuration>
+					<target>
+						<file>mirrortest-target-platform.target</file>
+					</target>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho.extras</groupId>
+				<artifactId>tycho-p2-extras-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<executions>
+					<!-- Mirror from the target platform - bundles are resolved from the 
+						target platform, which are resolved from the cache in ~/.m2/repository/p2 -->
+					<execution>
+						<id>mirror-from-target-platform</id>
+						<phase>compile</phase>
+						<goals>
+							<goal>mirror</goal>
```

Comment:
Why is a mirror goal here necessary at all? The `category.xml` should already "mirror" / create the repository here...